src/data/icons.json

```diff
@@ -5,8 +5,8 @@
       "color": "currentColor",
       "title": "",
       "desc": "",
-      "width": "var(--auro-size-lg)",
-      "height": "var(--auro-size-lg)",
+      "width": "var(--ds-size-300, 1.5rem)",
+      "height": "var(--ds-size-300, 1.5rem)",
```

Comment:
Why add the fallbacks here? We aren't doing that everywhere so why here?

Reply:
This will be released before anyone has updated to the new tokens. If we don't do this, icons will blow up.